src/SDKs/DataFactory/Management.DataFactory/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Resources;
+
+[assembly: AssemblyTitle("Microsoft Azure Data Factory Management Library")]
+[assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Data Factory Resources.")]
+[assembly: AssemblyVersion("0.0.0.0")]
```

Comment:
Assembly version cannot be 0.0.0.0 While you are in preview start with 0.2.0.0
Moving forward, bump up build version in AssemblyFileVersion to  0.2.1.0 and 0.2.2.0 and so on and leave AssemblyVersion at 0.2.0.0

Reply:
done